### ISSUE TITLE
Fix for `yarn zip` within Windows 10 using WSL2

### ISSUE THREAD

**Issue by michaelj916** [WorkshopCLI/workshop-cli#7]
# `yarn zip` the dist/ folder for deployment on Windows using WSL2
`yarn zip` fails to run in Windows 10 and it does not create any .zip files. I found a workaround to this using [WSL2 with Ubuntu](https://www.omgubuntu.co.uk/how-to-install-wsl2-on-windows-10). Since WSL2 supports a full Linux kernel and allows access between the Linux and Windows file structures, you can simply use WSL2 to run the `yarn zip` command. 
#### With Ubuntu, yarn is listed as `yarnpkg`, so you actually use `yarnpkg zip` within WSL2, but it works just fine. Make sure to do this within the base folder of your project.

# tl;dr commands within wsl2 : 
* sudo apt install yarnpkg
* yarnpkg global add @workshop-cli/workshop-cli
* cd your_project_folder/
* yarnpkg zip

**Comment by salexzee** [comment]
Hey @michaelj916! This is awesome and a great way to allow people to run the `yarn zip` command on a Win10 system.

I do still think it's necessary for me to provide a fix for people who don't want to use a Linux style terminal or those that don't understand the difference. I should be able to look into this more in about a week since I've finished a rebuild of my Win10 system and will be able to do some testing.

Thanks for looking into this. I usually don't prefer to write any code on Win, but WSL2 seems like a good way to bridge the gap between Unix style systems and Win systems. I'm going to mess around with it a bit.

**Comment by salexzee** [comment]
@michaelj916 I was finally able to hop on my Windows 10 gaming rig and get it setup for development. I was able to fix this issue within the workflow so now running `yarn zip` in either Powershell or Command Prompt works as initially intended. This was a workflow specific issue instead of a CLI issue so see https://github.com/WorkshopCLI/workshop-advanced-gulp/issues/24 for more information.

Thanks again for looking into WSL2! Closing this issue out.

**Comment by salexzee** [comment]
Make sure you update to the newest version of the CLI 0.0.11 to be able to create projects that have the fix in them.
